mscore/musescore.cpp

```diff
@@ -4948,13 +4955,28 @@ int main(int argc, char* av[])
             QString temp = parser.value("T");
             if (temp.isEmpty())
                    parser.showHelp(EXIT_FAILURE);
-            trimMargin = temp.toInt(); // ToDo: check whether conversion went OK, set to -1 if not
+            bool ok = false;
+            trimMargin = temp.toInt(&ok);
+            if (!ok)
+                  trimMargin = -1;
            }
       if (parser.isSet("x")) {
             QString temp = parser.value("x");
             if (temp.isEmpty())
                    parser.showHelp(EXIT_FAILURE);
-            guiScaling = temp.toDouble(); // ToDo: check whether conversion went OK, set to 1.0 if not
+            bool ok = false;
+            guiScaling = temp.toDouble(&ok);
+            if (!ok)
+                  guiScaling = 0.0;
```

Comment:
I sure wish I could get my hands on a system that fails to debug.  I do have a couple of TV sets and VGA adapters and so forth, I will see what I can do.  But I kind of suspect that in at least some cases, the hardware is simply not transmitting information about the physical size of the device, meaning there is no way to determine the true DPI value automatically, and so we will want o provide this option to allow the user to specify it.  But if can detect that we don't actually know the size, then at least we could probably pick a better default value for DPI as well - I just don't know what might make sense.